where_to_go/lib/models/dream_place.dart

```diff
@@ -0,0 +1,77 @@
+import "package:flutter/material.dart";
+import "../gen/assets.gen.dart";
+
+class Attraction {
+  final IconData icon;
+  final String label;
+
+  const Attraction({required this.icon, required this.label});
+}
+
+class DreamPlace {
+  final String name;
+  final String imagePath;
+  final String description;
+  final List<Attraction> attractions;
+
+  const DreamPlace({
+    required this.name,
+    required this.imagePath,
+    required this.description,
+    required this.attractions,
+  });
+}
+
+// I should probably move them to a separate file but i'm too lazy
+List<DreamPlace> exampleDreamPlaces = [
```

Comment:
```suggestion
final exampleDreamPlaces = [
```
you can also try `const` instead of final I don't think it will work with `Assets` shit